core/vm/actor/builtin/states/market_actor_state.hpp

```diff
@@ -37,9 +37,9 @@ namespace fc::vm::actor::builtin::states {
 
     using DealSet = adt::Set<UvarintKeyer>;
 
-    adt::Array<DealProposal> proposals;
-    adt::Array<DealState> states;
-    adt::Map<DealProposal, CidKeyer> pending_proposals;
+    adt::Array<DealProposal, 5> proposals;
```

Comment:
`kDefaultBitWidth`?

Reply:
No, `hamt::kDefaultBitWidth` is unrelated hamt's const.